test/conftest.py

```diff
@@ -1,9 +1,42 @@
 import pytest
+import requests
+from authnid.make_app import make_app
 import os
 
 
-@pytest.fixture(scope='module')
+@pytest.fixture
 def server_api():
-    host = os.getenv('SERVER_NAME')
-    server_name = f'https://{host}'
+    host = os.getenv("SERVER_NAME")
+    server_name = f"https://{host}"
     return server_name
+
+
+@pytest.fixture
+def request_client(server_api):
+
+    class RequestClient():
+
+        def __init__(self, server_api):
+            self.client = requests.Session()
+            self.server_api = server_api
+
+        def login(
+            self,
+            certs=(
+                "/app/ssl/client-certs/atat.mil.crt",
+                "/app/ssl/client-certs/atat.mil.key",
```

Comment:
Could these locations be specified in a way that is relative to where the tests are running, rather than hard coded based on the layout in the container?